src/tests/Sort.test.js

```diff
@@ -0,0 +1,46 @@
+const { app } = require('../pages/Application'); 
+
+
+describe('Inventory Sorting', () => {
+    before(async () => {
+        await app.login.navigate();
+        await app.login.performLogin('standard_user', 'secret_sauce');
```

Comment:
generally its a bad idea to have credentials hardcoded. It is not secure and if u're using the credentials in hundreds of tests if password changes u'll need to change them all. 
So, better way would be to hold them un a separately. For now separate file like user-data would be great